common/src/main/java/com/wynntils/core/functions/EnableableFunction.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.core.functions;
+
+public abstract class EnableableFunction<T> extends Function<T> {
+    public void init() {}
```

Comment:
Can we get Manager dependency registration here like features do?

Reply:
Probably. :) Do you want me to code it in right now, or do it later if/when the need actually arise?

Reply:
Ah this is a gonna need a bit more changes than I first thought. This is fine now, and I will probably add an interface for feature and functions to handle this.

Reply:
Like a `ModelDependant`? That's fine for me, as long as you give `Function` a default empty implementation, so trivial functions are not burdened with boilerplate.